git-cliff-core/src/changelog.rs

```diff
@@ -188,13 +191,13 @@ impl<'a> Changelog<'a> {
                 if let Some(version) = &release.version {
                     if skip_regex.is_some_and(|r| r.is_match(version)) {
                         skipped_tags.push(version.clone());
-                        trace!("Skipping release: {}", version);
+                        log::info!("Skipping release: {}", version);
```

Comment:
Wouldn't this be spammy if there are a lot of releases being skipped? Maybe consider debug instead.